### PR TITLE
Create the 2020 tutorial project.

### PR DESCRIPTION
**Pull request by zx100x100**
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/electrical_tutorials/1)

### CODE REVIEW

File: 2020_keychain_PIC/ee_tut/ee_tut.sch [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+F 3 "" H 8250 2350 50  0001 C CNN
+	1    8250 2350
+	1    0    0    -1  
+$EndComp
+$Comp
+L Device:R_US R4
+U 1 1 5F54130F
+P 6600 3750
+F 0 "R4" H 6668 3796 50  0000 L CNN
+F 1 "10k" H 6668 3705 50  0000 L CNN
```
Comment (akmorrison) [inline 2020_keychain_PIC/ee_tut/ee_tut.sch]: This seems a bit high for a current limiting resistor on an LED, is the part value intended?

File: 2020_keychain_PIC/ee_tut/sym-lib-table [hunk 2]
```diff
@@ -0,0 +1,3 @@
+(sym_lib_table
+  (lib (name New_Library)(type Legacy)(uri C:/Users/zx100/Documents/Rocketry/electrical_tutorials/2020_keychain_PIC/ee_tut/New_Library.lib)(options "")(descr ""))
```
Comment (akmorrison) [inline 2020_keychain_PIC/ee_tut/sym-lib-table]: Absolute path, should be `(uri ${KIPRJMOD}/New_Library.lib)`

File: 2020_keychain_PIC/ee_tut/ee_tut.sch (hunk 1 above)
Comment (zx100x100) [reply 2020_keychain_PIC/ee_tut/ee_tut.sch]: lol whoops, went a little over zealous on labeling things as 10K

**Review by dwkletke — COMMENTED** [review]
Honestly I appreciate the symmetry of this layout. Overall a really pretty board.

I am not sure if you meant to do this for the tutorial, but you are daisy-chaining R1 and R2 with their respective switches. 
image
image

In addition, R4 is daisy-chained with the PIC for ICSPCLK (again unsure if this is intended for the tutorial).
image

Also nitpick: MCLR line feels real close to R5 and D2 for me, but probably not an issue.
image

**Review by JaredWatson — CHANGES_REQUESTED** [review]
Overall looks good! I think this is a great first project. Edit: The phrasing made it sound like this was your first project haha, I meant for a first year :joy:

My comments:

- Like Dawson mentioned a number of your traces are daisy chained (because the function of the line relies on the part they're chained through) but after discussing it on slack the agreement was replacing the part would be easier without the daisy chains, so those should be removed

- SW3 is a double throw switch, but only one of the throws is being used (like you mentioned on slack). Marking the useful switch with it's function and potentially marking the unused throw would be useful to a first time reader

- A small aesthetic nitpick that needs not be addressed, but the symmetry of the battery holder is not in line with the keyring hole :(
image

**Comment by AidanHa** [comment]
I'm not a reviewer, but I took a quick look at the board:
I think if you address the comments above (ie daisychain), then I think it looks good.
For the schematic, it felt weird not having the actual voltage (and just having `+batt`) might be just me tho.

This is honestly great! I wish I had this when I was starting out

**Comment by QuantumManiac** [comment]
Looks like it'll be good to go once the previous concerns are addressed. Do we want to reduce the clearance of the pours? Looks It's currently at 20 mils.

Super cool though, definitely something I'd fidget with if I had one on my key-ring.

**Comment by zx100x100** [comment]
I have addressed y'all's awesome comments, some notes:
Dawson: I'm leaving ICSP dasy-chained through the PIC, the routing gets notably messier if I don't and I don't mind daisy-chaining through critical pads (If that pad is broken the board won't work any more regardless.)

Jared: I modded the schematic so it aligns with the fact that it is a SPDT switch with only 2 use pins, I didn't make any note on the silkscreen.

Thanks! :party_blob:

**Review by dwkletke — APPROVED** [review]
Looks good to me!

**Review by QuantumManiac — APPROVED** [review]
LGTM!

**Review by JaredWatson — APPROVED** [review]
LGTM! :party_blob:

**Comment by zx100x100** [comment]
Alright, looks like everything is good, rebasing and merging.

**Pull request merged**